Classes/Controller/CacheWarmupController.php

```diff
@@ -276,7 +276,7 @@ public function fetchSitesAction(): ResponseInterface
 
             $sitemapsFound = false;
             $action = [
-                'title' => $site->getConfiguration()['websiteTitle'] ?: BackendUtility::getRecordTitle('pages', $row),
+                'title' => $site->getConfiguration()['websiteTitle'] ?? BackendUtility::getRecordTitle('pages', $row),
```

Comment:
Thanks, @josefglatz! Nice catch 🙂 I'd like to adapt it a bit to handle empty strings as well:

```suggestion
                'title' => ($site->getConfiguration()['websiteTitle'] ?? null) ?: BackendUtility::getRecordTitle('pages', $row),
```

Reply:
👍